Index: linux-custom/drivers/usb/gadget/ether.c
===================================================================
--- linux-custom.orig/drivers/usb/gadget/ether.c
+++ linux-custom/drivers/usb/gadget/ether.c
@@ -237,8 +237,7 @@ static int __init rndis_do_config(struct
 		c->bmAttributes |= USB_CONFIG_ATT_WAKEUP;
 	}
 
-	return rndis_bind_config(c, hostaddr, cpu_to_le16(CDC_VENDOR_NUM),
-				 strings_dev[STRING_MANUFACTURER_IDX].s);
+	return rndis_bind_config(c, hostaddr);
 }
 
 static struct usb_configuration rndis_config_driver = {
